include/albatross/models/gp.h

```diff
@@ -411,17 +381,16 @@ class GaussianProcessRegression
     : public GaussianProcessBase<CovFunc, GaussianProcessRegression<CovFunc>> {
 public:
   using Base = GaussianProcessBase<CovFunc, GaussianProcessRegression<CovFunc>>;
+  using Base::Base;
```

Comment:
This pulls forward the constructors from `Base` in place of defining constructors for `GaussianProcessRegression` that just call the equivalent constructor in `Base`?

Reply:
Yeah exactly.

Reply:
Nice, didn't know that this could be done so elegantly!